docs/general/ad-filtering/create-own-filters.md

```diff
@@ -1087,12 +1087,16 @@ Basic URL exceptions shall not disable rules with `$hls` modifier. They can be d
 
 :::
 
-**Restrictions**
+:::caution Restrictions
```

Comment:
If we go with this, restrictions should be placed in the bottom of a modifier description.

Let's use a simple plan:

Description and syntax
Examples
Restrictions
Compatibility